Source/HelixToolkit.Maths/BoundingBoxHelper.cs

```diff
@@ -143,5 +143,9 @@ public static BoundingBox Transform(this BoundingBox box, Matrix4x4 transform)
 
             return new BoundingBox(min, max);
         }
+        public static RectangleF Translate(this RectangleF rect, Vector2 translation)
```

Comment:
I don't think this function belongs to BoundingBoxHelper. Should be in RectangleF helper or simply put it inside the RectangleF.

Reply:
Based on input, since it's the extension method of the input type, not the output type.